src/gtimelog/main.py

```diff
@@ -2,8 +2,8 @@
 import sys
 import time
 
-
-DEBUG = '--debug' in sys.argv
+from gtimelog import DEBUG
+from gtimelog.email import MAIL_PROTOCOLS, EmailError, send_email
```

Comment:
There should be the minimum possible amount of code before the first `mark_time()` invocation.

`from gtimelog import DEBUG` needs to be there, but `from gtimelog.email import anything` shouldn't be.